doc/api/next_api_changes/deprecations.rst

```diff
@@ -415,3 +415,8 @@ parent figure, though they can be later removed with ``ax.remove()``.
 
 ``.BboxBase.inverse_transformed`` is deprecated (call `.BboxBase.transformed`
 on the `~.Transform.inverted()` transform instead).
+
+``pie(normalize=None)``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+``normalize=None`` gives the current behavior and if ``sum(x) < 1``, pass ``normalize=False`` 
```

Comment:
Please try to phrase this so that it's understandable without further context. Something like

> pie() used to draw a partial pie if the sum of the values was < 1. This behavior is deprecated and will change to always normalizing the values to a full pie by default. If you want to draw a partial pie, please pass `normalize=False` explicitly.